docs/installation.rst

```diff
@@ -211,6 +197,25 @@ You're finally ready to install Keylime::
     sudo python setup.py install
 
 
+Configuring basic (m)TLS setup
+------------------------------
+Keylime uses mTLS authentication between the different components.
+By default the verifier creates a CA for this under :code:`/var/lib/keylime/cv_ca/` on first startup.
+This consists mainly of three different components:
```

Comment:
The directory contains files for three different components:

Reply:
fixed